src/features/defi/components/GetStarted/GetStarted.tsx

```diff
@@ -0,0 +1,71 @@
+import { Box, Flex } from '@chakra-ui/layout'
+import { Button, VStack } from '@chakra-ui/react'
+import osmosis from 'assets/osmosis.svg'
+import { Text } from 'components/Text'
+
+import { DefiModalHeader } from '../DefiModalHeader/DefiModalHeader'
+
+type GetStartedProps = {
+  assetId: string
+}
+
+const CAIP19_TO_MAX_APR = {
```

Comment:
two things:

* can we rename this to assetId rather than caip19, that's my fault for starting this antipattern leaking the implementation details for now we derive asset ids
* what's the long term plan for this? this needs a todo and to come from some service.

Reply:
Can't it be colocated for now? We're not going to have so many members in that mapping for now so IMO it's fine to have it here until it starts to get boilerplaty

Reply:
yup, can be for now, let's rename it to `ASSET_ID_TO_MAX_APR` though so my antipattern doesn't spread